src/main/java/com/revature/controllers/ReviewController.java

```diff
@@ -0,0 +1,141 @@
+package com.revature.controllers;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.servlet.http.HttpSession;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+
+import com.revature.annotations.Authorized;
+import com.revature.dtos.ReviewRequest;
+import com.revature.models.Review;
+import com.revature.models.User;
+import com.revature.services.ReviewService;
+
+@RestController
+@RequestMapping("/api/review")
```

Comment:
This should be just `/review` since `/api` is already the root endpoint of the API

Reply:
It is not, our only other api endpoint follows the same pattern in ProductController

Reply:
Oh I see